idea_town/frontend/templates/idea_town/frontend/index.html

```diff
@@ -3,6 +3,10 @@
 <head>
     <title>Idea Town</title>
     <link rel="stylesheet" href="{{ static('styles/main.css') }}">
+    <link rel="x-api-root" href="{{ url('api-root') }}?format=json">
+    <link rel="x-logout" href="{{ url('account_logout') }}">
+    {# TODO: Make this a dev / prod setting? Or, expose in an API endpoint? #}
+    <link rel="x-addon" title="Idea Town" href="https://github.com/mozilla/idea-town-addon/blob/master/dist/idea-town-addon-0.0.1.xpi?raw=true">
```

Comment:
Well, that touches settings & env vars & docker setup, so I was being lazy :) But, I can circle back to that now